### DISCUSSION TITLE
Examples for Small/Rubble/Blast_Mark_0

### DISCUSSION THREAD

**Discussion by Jdance-Media** [SmartlyDressedGames/Unturned-3.x-Community#3863] [Q&A] [answered]
Third post in one day (please don't kill me, lol). Blast marks appear to be 'Small' objects that partially function as decals. For me, they are incredibly useful, especially if there are two contrasting colors that I need to darken (ex. modern/vanilla objects on a WW2 map). Couple others have been trying to replicate it, but still no luck. Figured I mind as well give it a try and ask for an example.

**Comment by SDGNelson** [comment, marked as answer]
Sorry, unfortunately these are not moddable (yet). They have the Decal script attached in Unity. In the future I intend to make decals data driven so that the bullet impact decals can be modded, and other cases like this.

**Reply by Jdance-Media** [reply]
👍, understandable.
